docs/upgrade.html

```diff
@@ -102,6 +102,9 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         </li>
                         <li>The <code>--topic-white-list</code> option was removed from the <code>kafka-replica-verification</code> command line tool.
                             Please use <code>--topics-include</code> instead.
+                        </li>li>
```

Comment:
@FrankYang0529 @chia7712 Seems there is a mistake here with tag `</li>li>` => `</li>`, shall we correct it please?

Reply:
I will create a PR for it. Sorry for that.

Reply:
PR: https://github.com/apache/kafka/pull/18032